common/src/main/java/org/opensearch/ml/common/memorycontainer/MemoryType.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.memorycontainer;
+
+/**
+ * Enum representing the type of memory entry
+ */
+public enum MemoryType {
+    RAW_MESSAGE("RAW_MESSAGE"),
+    FACT("FACT");
```

Comment:
https://github.com/mem0ai/mem0/blob/7d1d0ca8063d03e58265dacd7246229696b1260e/mem0/configs/enums.py#L4

What do you think about following this?